src/osx/carbon/statbrma.cpp

```diff
@@ -78,42 +78,22 @@ void wxStatusBarMac::InitColours()
 {
     if ( WX_IS_MACOS_AVAILABLE(10, 14) )
     {
-        // FIXME: None of this is correct and is only very loose
-        //        approximation. 10.14's dark mode uses dynamic colors that
-        //        use desktop tinting. The only correct way to render the
-        //        statusbar is to use windowBackgroundColor in a NSBox.
-        wxColour bg = wxSystemSettings::GetColour(wxSYS_COLOUR_APPWORKSPACE);
-        m_textActive = wxSystemSettings::GetColour(wxSYS_COLOUR_BTNTEXT);
-        m_textInactive = wxSystemSettings::GetColour(wxSYS_COLOUR_GRAYTEXT);
-
         if ( wxSystemSettings::GetAppearance().IsDark() )
         {
-            // dark mode appearance
-            m_textActive = wxColour(0xB0, 0xB0, 0xB0);
-            m_bgActiveFrom = wxColour(0x32, 0x32, 0x34);
-            m_bgActiveTo = wxColour(0x29, 0x29, 0x2A);
-            m_borderActive = wxColour(0x00, 0x00, 0x00);
-            m_borderInactive = wxColour(0x00, 0x00, 0x00);
+            m_textActive = wxColour(0xA9, 0xA9, 0xA9);
+            m_textInactive = wxColour(0x67, 0x67, 0x67);
```

Comment:
Do we still have no way to get these colours from the system?

Reply:
I don't know which exactly, but we if we expose this (what would be the best way ? constants ? enums )

https://developer.apple.com/design/human-interface-guidelines/macos/visual-design/color#system-colors

we should be able to find a match

Reply:
I don't think these colors are among system colors (no reason to: this is OS-rendered chrome), and certainly not with matching *semantics*. The only one semantically close is `windowFrameTextColor`, which doesn't match (and doesn't handle inactivated state).